week_3/15_apr/strings.js

```diff
@@ -0,0 +1,15 @@
+let email = 'kcolaco'
+console.log(email)
+
+let firstName = 'Kevin'
+let lastName = 'Colaco'
+
+let fullName = firstName + ' ' + lastName
```

Comment:
avoid using + to concat the strings use.... ${firstName} {lastName} with ``.

Reply:
@abhi-codal Yes this was for practice. I have used the ${} in successive files.